.circleci/config.yml

```diff
@@ -535,10 +547,17 @@ jobs:
 workflows:
   main:
     jobs:
-      - save-start-time
-      - install:
+      - save-start-time:
+          filters:
+            branches:
+              only: master
+      - waiter:
           requires:
             - save-start-time
+      - send-stats:
+          requires:
+            - waiter
+      - install
```

Comment:
Shouldn't install still depend on save-start-time? Otherwise this could be problematic in case there are not enough runners available.

Reply:
When I make install depend on save-start-time, then it (and the whole pipeline) won't always run because save-start-time only runs in master